### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by jorshali** [jorshali/snowflake-pool#4] [Ideas]
@jacobworrel We can begin brainstorming the future roadmap here.  Just add any ideas you have and we can discuss and prioritize.

**Comment by jacobworrel** [comment]
@jorshali I wanted to start by writing some tests since I noticed there aren't any right now, but I'm having trouble figuring out what to test since this is just a wrapper around `generic-pool`'s `createPool` method. Any ideas?

**Reply by jorshali** [reply]
@jacobworrel yes, that’s why I didn’t include any tests in the initial implementation.  If you have some additional features to add, tests can be included for the new features.
